doc/api/packages.md

```diff
@@ -612,16 +612,22 @@ The following condition definitions are currently endorsed by Node.js:
 * `"browser"` - any environment which implements a standard subset of global
   browser APIs available from JavaScript in web browsers, including the DOM
   APIs.
+* `"deno"` - indicates a variation for the Deno platform.
 * `"development"` - can be used to define a development-only environment
   entry point. _Must always be mutually exclusive with `"production"`._
 * `"production"` - can be used to define a production environment entry
   point. _Must always be mutually exclusive with `"development"`._
+* `"source"` - indicates the original .js source file without minification
+  or bundling optimizations, useful for development and debugging workflows.
+* `"types"` - can be used by typing systems to resolve the typing file for
+  the given export, possible since the interface should be the same for all
+  variations.
 
 The above user conditions can be enabled in Node.js via the
 [`--conditions` flag][].
 
-Platform specific conditions such as `"deno"`, `"electron"`, or `"react-native"`
-may be used, but while there remain no implementation or integration intent
+Platform specific conditions such as `"electron"`, or `"react-native"` may
+be used, but while there remain no implementation or integration intent
 from these platforms, the above are not explicitly endorsed by Node.js.
```

Comment:
Out of scope for this PR perhaps, so feel free to ignore, but [our style guide](https://github.com/nodejs/node/blob/master/doc/guides/doc-style-guide.md) refers folks to the [Microsoft Writing Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/) for things not covered in our style guide. Microsoft style would be more like this:

```suggestion
from these platforms, we do not recommend them.
```

Refs: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/recommend
Refs: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above